### PR TITLE
Prepare 0.20.0 release

### PR DESCRIPTION
**Pull request by dyladan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,46 @@
...
+### 💡 Enhancements 💡
+
+- `statsdreceiver`: Adapt statsd receiver example config to remove filtering of histogram metrics (#385)
+- `tailsamplingprocessor`: Adds decision cache for non-sampled trace IDs ([#31583](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31583))
+
+### 🧰 Bug fixes 🧰
+
+- `service`: ensure OTLP emitted logs respect severity ([#11718](https://github.com/open-telemetry/opentelemetry-collector/pull/11718))
+- `processor/k8sattribute`: fixes parsing of k8s image names to support images with tags and digests. ([#36131](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36131))
+
```
Comment (evan-bradley) [inline CHANGELOG.md:45]:
```suggestion
<summary>Highlights from the upstream Collector changelog</summary>

### 🛑 Breaking changes 🛑

- `otelcol`: Change all logged timestamps to ISO8601. ([#10543](https://github.com/open-telemetry/opentelemetry-collector/pull/10543))
This makes log timestamps human-readable (as opposed to epoch seconds in
scientific notation), but may break users trying to parse logged lines in the
old format.
- `k8sattributesprocessor`: Move k8sattr.fieldExtractConfigRegex.disallow feature gate to Beta. ([#25128](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/25128))
Disable the k8sattr.fieldExtractConfigRegex.disallow feature gate to get the old behavior.

### 💡 Enhancements 💡

- `statsdreceiver`: Adapt statsd receiver example config to remove filtering of histogram metrics (#385)
- `tailsamplingprocessor`: Adds decision cache for non-sampled trace IDs ([#31583](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31583))

### 🧰 Bug fixes 🧰

- `service`: ensure OTLP emitted logs respect severity ([#11718](https://github.com/open-telemetry/opentelemetry-collector/pull/11718))
- `processor/k8sattribute`: fixes parsing of k8s image names to support images with tags and digests. ([#36131](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/36131))

</details>
```
We should move the upstream specific ones to the upstream changelog section, and I don't think we have any distro-specific updates this time, so we can just remove that section.

Comment (dyladan) [reply CHANGELOG.md:45]: oops. thanks

**Comment by dyladan** [comment]
@evan-bradley any idea how i can get the changelog to skip or pass? It's not exactly relevant for this PR right?

**Comment by evan-bradley** [comment]
You can add the `Skip Changelog` label or prepend the PR's title with `[chore]` for PRs that don't need a changelog entry.

**Review by arminru — APPROVED** [review]

**Comment by dyladan** [comment]
ah ok. i was looking at the 0.19.0 release prep and didn't see a label so i wasn't sure what was needed. thanks

**Pull request merged**